docs/framework/interop/configure-net-framework-based-com-components-for-reg.md

```diff
@@ -46,7 +46,7 @@ Registration-free activation for .NET Framework-based components is only slightl
     <assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">  
     ```  
   
-     For information about manifest elements and their attributes, search for "Application Manifests Reference" in the MSDN Library.  
+     For information about manifest elements and their attributes, search for "Application Manifests Reference" in Microsoft Docs.  
```

Comment:
this is not very helpful. Let's change this to 
`search for "Application Manifests Reference" in Microsoft Docs.`
to
`see [Application Manifests](https://msdn.microsoft.com/library/windows/desktop/aa374191.aspx).`